src/drive/lib/encryption.js

```diff
@@ -17,6 +18,9 @@ export const isEncryptedFile = file => {
 }
 
 export const getEncryptionKeyFromDirId = async (client, dirId) => {
+  if (flag('drive.enable-encryption')) {
+    return null
+  }
```

Comment:
let's update the changelog also 🙏

Reply:
Sorry, I will eventually think of this... Maybe an automatic reminder in the PR would be useful?

Reply:
https://github.com/cozy/cozy-drive/pull/2541

Reply:
so bad memory, or so happy with automatisation ^^